ui-kit/GroupCard/GroupCard.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { systemPropTypes } from '..';
+import { Box, Avatar, SquareAvatar, Button, Icon } from '..';
+import Styled from './GroupCard.styles';
+import { textTrimmer } from '../../utils';
+
+const GroupCard = (props = {}) => {
+  const summaryLength = props.summary ? props.summary.length : 0;
+  const maxChar = 130;
+
+  // Option if we want to add a Avatar Count label for leaders
+  // const heroAvatarsDiff = props.totalHeroAvatars - props.heroAvatars.length;
+  const avatarsDiff = props.totalAvatars - props.avatars.length;
+
+  return (
+    <Styled display="flex" flexDirection="column" {...props}>
```

Comment:
Gotcha yeah that makes sense! In `GroupCard.styles` I added a `GroupCard` style that overrides the default `Card` with `styled(Card)` to fix that. Then in the `GroupCard` component instead of `Styled` I'm now using `Styled.GroupCard`. Let me know if that works! Sorry still getting the hang of `styled-components` 😅